cfpb/cfgov-refresh
# Adjust Ask breadcrumbs to lead to portal pages.

Currently, Ask CFPB breadcrumbs on answer and category pages lead back
to categories and the ask-cfpb root.

When we decommission category and subcategory pages, breadcrumbs should
lead back to portal pages instead, falling back to see-all pages if necessary.

## Testing
This commit has been deployed to the internal DEV3 server for review.

You can also check breadcrumbs locally:

First, turn the new breadcrumbs on by activating the ASK_CATEGORIES_OFF flag: 
1. In the Wagtail admin, go to **Settings --> Flags**
2. Scroll to the bottom and click on ADD A CONDITION
3. Select Flag ASK_CATEGORIES_OFF
4. Set Condition name to **boolean** and Expected value to **True**
5. Hit SAVE NEW CONDITION

Then check some answer page breadcrumbs:

Answer page with live portal has breadcrumb back to portal:
- http://localhost:8000/ask-cfpb/i-owe-more-on-my-current-loan-than-my-current-vehicle-is-worth-what-do-i-need-to-know-if-i-buy-a-new-vehicle-en-779/

Page with no live portal points to its see-all topic page:
- http://localhost:8000/ask-cfpb/what-is-a-payday-loan-en-1567/

Spanish pages (none have live portals) point to see-all page:
- http://localhost:8000/es/obtener-respuestas/debo-mas-en-mi-prestamo-de-lo-que-vale-mi-vehiculo-que-debo-saber-si-compro-un-vehiculo-nuevo-es-779/
- http://localhost:8000/es/obtener-respuestas/me-gustaria-poder-tener-la-ayuda-de-mis-amigos-o-familiares-con-el-pago-de-mis-facturas-y-actividades-bancarias-cuales-son-mis-opciones-es-1145/

## Notes
- This puts the switch behind a feature flag, `ASK_CATEGORIES_OFF`
- I moved one-off scripts that have been used to `/deprecated/`, for later removal.
